sklearn/ensemble/tests/test_bagging.py

```diff
@@ -748,3 +749,51 @@ def test_set_oob_score_label_encoding():
     x3 = BaggingClassifier(oob_score=True,
                            random_state=random_state).fit(X, Y3).oob_score_
     assert_equal([x1, x2], [x3, x3])
+
+
+def test_bagging_pipeline_with_sparse_inputs():
+    # Check that BaggingRegressor can accept sparse pipelines inputs
```

Comment:
What you're testing with is not what we call sparse. I presume sparse inputs are already tested too. Do you mean to test nans and multioutput (which probably belongs in a separate test function)?